Tests/TuistGeneratorTests/Generator/ProjectFileElementsTests.swift

```diff
@@ -585,6 +585,36 @@ final class ProjectFileElementsTests: XCTestCase {
                                                      sourceRootPath: AbsolutePath("/a/b/c/project"))
         XCTAssertEqual(got, RelativePath("../../../framework"))
     }
+
+    func test_generateDependencies_sdks() throws {
+        // Given
+        let pbxproj = PBXProj()
+        let project = Project.test()
+        let sourceRootPath = AbsolutePath("/a/project/")
+        let groups = ProjectGroups.generate(project: project,
+                                            pbxproj: pbxproj,
+                                            sourceRootPath: sourceRootPath)
+
+        let sdk = try SDKNode(name: "ARKit.framework", status: .required)
+
+        // When
+        try subject.generate(dependencies: [sdk],
+                             path: sourceRootPath,
+                             groups: groups, pbxproj: pbxproj,
+                             sourceRootPath: sourceRootPath,
+                             filesGroup: .group(name: "Project"))
+
+        // Then
+        XCTAssertEqual(groups.frameworks.debugChildPaths, [
```

Comment:
What is `debugChildPaths`? The name is a bit misleading.

Reply:
Naming is hard :) ...

It's a [convince method](https://github.com/tuist/tuist/blob/master/Tests/TuistGeneratorTests/Generator/ProjectFileElementsTests.swift#L600) I added a while back to help test group hierarchies in a simple concise manner.

Open to better name suggestions.

Reply:
What about `flattenedChildren`?

Reply:
I like it 👍 - will update